homeassistant/components/yale_smart_alarm/manifest.json

```diff
@@ -2,7 +2,8 @@
   "domain": "yale_smart_alarm",
   "name": "Yale Smart Living",
   "documentation": "https://www.home-assistant.io/integrations/yale_smart_alarm",
-  "requirements": ["yalesmartalarmclient==0.1.6"],
-  "codeowners": [],
+  "requirements": ["yalesmartalarmclient==0.3.2","backoff==1.10.0"],
```

Comment:
Please add links to the openings post of this PR with links to release notes and dif compare views of the upstream library changes (as they need to be checked as well).

Reply:
https://github.com/domwillcode/yale-smart-alarm-client/pull/27 @frenck  will the closed PR cover what you're asking for ?

Reply:
> [domwillcode/yale-smart-alarm-client#27](https://github.com/domwillcode/yale-smart-alarm-client/pull/27) @frenck will the closed PR cover what you're asking for ?

I will put the links as requested above. However it seems the addition of backoff he did manually as the github package still says 3.1 but then at least 3.1 to 3.2 was only the addition of backoff requirement in the manifest file.

Reply:
Update this PR description to contain at least one (or multiple) of the following:

- A link to the release notes of this package version, and all versions in between.
- A link to the changelog of this package.
- A link to a Git(Hub) diff/compare view from the current version to the bumped version.

This allows us to review upstream changes, which is needed to decide this change is working as intended.